tests/test_api.py

```diff
@@ -772,22 +756,30 @@ def test_get_collection_item_json_ld(config, api_):
     assert rsp_headers['Content-Language'] == 'en-US'
     feature = json.loads(response)
     assert '@context' in feature
-    assert feature['@context'][
-        0] == 'https://geojson.org/geojson-ld/geojson-context.jsonld'
+    assert all((f in feature['@context'][0] for
+                f in ('schema', 'type', 'geosparql')))
     assert len(feature['@context']) > 1
     assert 'schema' in feature['@context'][1]
     assert feature['@context'][1]['schema'] == 'https://schema.org/'
-    assert feature['properties']['stn_id'] == '35'
+    assert feature['stn_id'] == '35'
     assert feature['id'].startswith('http://')
     assert feature['id'].endswith('/collections/obs/items/371')
     expanded = jsonld.expand(feature)[0]
     assert expanded['@id'].startswith('http://')
     assert expanded['@id'].endswith('/collections/obs/items/371')
-    assert expanded['https://purl.org/geojson/vocab#properties'][0][
-        'https://schema.org/identifier'][0][
+    assert expanded['https://schema.org/identifier'][0][
             '@type'] == 'https://schema.org/Text'
-    assert expanded['https://purl.org/geojson/vocab#properties'][0][
-        'https://schema.org/identifier'][0]['@value'] == '35'
+    assert expanded['https://schema.org/identifier'][0][
+            '@value'] == '35'
+    assert expanded['http://www.opengis.net/ont/geosparql#hasGeometry'][0][
+            'http://www.opengis.net/ont/geosparql#asWKT'][0][
+            '@value'] == 'POINT (-75 45)'
+    assert expanded['https://schema.org/geo'][0][
+            'https://schema.org/latitude'][0][
+            '@value'] == 45.0
+    assert expanded['https://schema.org/geo'][0][
+            'https://schema.org/longitude'][0][
+            '@value'] == -75.0
```

Comment:
Is it possible to do tests for non-point geometries, too?

Reply:
added tests for LineString, Polygon, MultiPoint, MultiLineString, and MultiPolygon